src/viewer.js

```diff
@@ -876,6 +881,38 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
         return this;
     },
 
+    /**
+     * Opens a collection of tiled images in the viewer.
+     * @function
+     * @param {Array|String} collectionSources - A single URL or an array of URLs to the collection sources (DZC files).
+     * @returns {OpenSeadragon.Viewer} Chainable.
+     * @fires OpenSeadragon.Viewer.event:open
+     * @fires OpenSeadragon.Viewer.event:open-failed
+     */
+    openCollection: function (collectionSources, batch = 300) {
+        //var _this = this;
+        if (!$.isArray(collectionSources)) {
+            collectionSources = [collectionSources];
+        }
+
+        // use createCollectionSource
+        $.createCollectionSource({
+            url: collectionSources[0],
+            ajaxWithCredentials: this.ajaxWithCredentials,
+            ajaxHeaders: this.ajaxHeaders,
+            success: function(tileSources) {
+               this.open(tileSources, this.initialPage);
+            }.bind(this),
+            error: function(err) {
+                if (this.error) {
+                    this.error(err);
+                }
+            }.bind(this)
+        });
+
+        return this;
```

Comment:
I know `open` returns `this` to be chainable, but I'm trying to move us away from that style of API. Let's leave it empty here.